packages/lib-react-components/src/Markdownz/Markdownz.stories.js

```diff
@@ -23,25 +75,38 @@ export default {
   }
 }
 
-export const Default = () => <Markdownz>{markdownExample}</Markdownz>
+export const Default = () => (
+  <>
+    <MarkdownExamplesTable />
+    <Markdownz>
+      {markdownExample}
+    </Markdownz>
+  </>
+)
 
 export const InProjectContext = () => (
-  <Markdownz projectSlug='zooniverse/snapshot-wakanda'>
-    {markdownExample}
-  </Markdownz>
+  <>
+    <MarkdownExamplesTable projectSlug='zooniverse/snapshot-wakanda' />
+    <Markdownz>
+      {markdownExample}
+    </Markdownz>
+  </>
 )
 
 export const WithCustomComponents = () => (
-  <Markdownz components={{ tr: TableRowWithBorder }}>
-    {markdownExample}
-  </Markdownz>
+  <>
+    <MarkdownExamplesTable components={{ tr: TableRowWithBorder }} />
```

Comment:
Please run lib-react-components storybook locally and double check. I do not see TableRowWithBorder` applied as a custom component.

Reply:
I see borders on each row in the markdown table, as expected.

Reply:
That's the table in the last example, by the way. The table that's used to lay out the page is written in JSX, not markdown, so won't change.

Reply:
<img width="600" alt="The markdown table example from the storybook, but each row has a custom bottom border." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/42858992-c363-44ef-ad2a-56495f821784">

This is in Firefox, by the way, but it shouldn't vary at all between browsers.

Reply:
Is it not the borders on the entire Markdownz layout table?

![Screenshot 2023-10-25 at 6 56 52 PM](https://github.com/zooniverse/front-end-monorepo/assets/23665803/0d610247-1b54-4def-9d18-c29cf6818430)

Reply:
^ my screenshot is from the [deployed](https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/components-markdownz--with-custom-components) storybook

Reply:
So to be clear, the difference between the "old" Markdownz storybook examples and the "new" examples is that the new examples are isolated? For example, the footnotes do not get bumped all the way down to the bottom of the story anymore. The footnotes just exist is their own square. 

I think this is fine! I was confused on how to compare changes to the story examples

Reply:
You’re right that this is a confusing story. I’ll move the table to the top of the examples, so that it’s  more obvious what’s changed. 

I forgot to note that I replaced the original story with individual markdown examples. I found that the current story is hard to debug. If there’s a problem anywhere in the markdown, the whole story crashes. Isolated examples made debugging easier.